masonry/src/core/contexts.rs

```diff
@@ -730,12 +730,19 @@ impl_context_method!(
             self.widget_state.size
         }
 
-        // TODO - Remove? A widget doesn't really have a concept of its own "origin",
-        // it's more useful for the parent widget.
-        /// The layout rect of the widget.
+        /// The layout rect of the widget in window coordinates.
         ///
-        /// This is the layout [size](Self::size) and origin (in the parent's coordinate space) combined.
-        pub fn layout_rect(&self) -> Rect {
+        /// This is the layout [size](Self::size) and [window_origin](Self::window_origin) combined.
+        ///
+        /// See [layout rect documentation](crate::doc::doc_06_masonry_concepts#layout-rect)
+        /// for details.
+        pub fn global_layout_rect(&self) -> Rect {
+            Rect::from_origin_size(self.widget_state.window_origin(), self.widget_state.size)
+        }
```

Comment:
I'm not sure whether this is really useful at all, as this doesn't really represent the (global) widgets position, bounds or size (due to potential transformations).

I wonder whether we should get rid of "layout rects"?
I think they cause more confusion than being helpful.